packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.scss

```diff
@@ -1,3 +1,22 @@
-.dropdown {
+.dropdownWrapper {
+	position: relative;
+}
+
+.popover {
 	width: 320px; // TODO: Remove when width sizing management has been covered
+	max-width: none;
+	padding: 0;
```

Comment:
Those (max-width and padding) don't seem to be used.

Reply:
Those 2 properties has been globally overridden in a component style :D

/packages/components/src/RichTooltip/RichTooltip.scss
```javascript
:global(.popover) {
	padding: 0;
	border: $tc-popover-border-width solid $tc-popover-border-color;
	border-radius: $border-radius-base;
	background-color: $white;
	box-shadow: $tc-popover-box-shadow;
	max-width: $tc-popover-max-width;
}
```

So I think it's useful if the other component don't override globally the theme. I have discussed with @jare-talend, We should discuss it now with the UX/designers to eventually find a way to have a common style base for the dropdown/popover. But it's far away my priorities.

Reply:
Ok but that doesn't change the fact those lines are not applied so you can remove them.

Reply:
> Ok but that doesn't change the fact those lines are not applied

Yes ! (In the current time for the current build)

> so you can remove them.

I can bu I don't want ! :p


This is no about _what actually is_ but about _what should be done_. I want my dropdown like that and no common style is done for that so I do it.
I will not rely on an other component for setting the style I want for multiples reasons :
- When it'll be patched, the current component will broke
- I'm relying on an other component without any explicit mention, in any of the two components. So if the other component decide to tweak a little some padding or whatever (without patching the global style), the current component will be impacted without knowing it.
- If an App decide to not include/load the other component, the style will be different than expected in the storybook.

Those are style properties **my component needs to have** unless we decide to have a common dropdown/popover style in theme for it.
I'll definitely not rely on an other component with an implicit totally invisible pernicious dependency.